python/tests/Examples.py

```diff
@@ -13,6 +14,15 @@
 # Running these tests will output the examples to the console
 class Examples(unittest.TestCase):
 
+    test_dir_path = os.environ.get("DIR_PATH", None)
```

Comment:
I would move setting this directory variable down in the main method below. As you mentioned, you could then check to see if the path exists and warn the user if the directory has not been created yet.